wire_response.go

```diff
@@ -56,6 +73,7 @@ func (s *WireQueryResponse) UnmarshalJSONTyped(in []byte, typed bool) error {
 	if err := json.Unmarshal(in, &aux); err != nil {
 		return err
 	}
+	bypass, _ := s.decodeDataWithNumber(in)
```

Comment:
I am distrustful of disregarding this error. It looks like we can get here if we get an error calling the decoder, or if we didn't get a []interface{}, and i don't know whether those can happen, but... I feel like we should handle it Just In Case.

Reply:
it can happen in tests, but i'll clear up a bit